### PR TITLE
Implementing CLI interface for identifier

### LINKED ISSUE #21: Implement CLI interface

### PR DESCRIPTION
**Pull request by TB-DevAcc**
Closes #21

### CODE REVIEW

**Comment by TB-DevAcc** [comment]
Currently works non-asynchronously like this:

SHBcli1

SHBcli2

Command chaining is also possible e.g.:
```
shddeviceidentifier collect read identify
```

**Comment by jo-jstrm** [comment]
Looks great!

Why the import in `__init__.py`?

File: device-identifier/shbdeviceidentifier/app.py [hunk 1]
```diff
@@ -1,26 +1,235 @@
...
+@pass_ctx
+@logger_wraps()
+def identify(ctx, file_path, out):
+    """
+    Identifies a device.
+    """
+    file_path = check_file_path(ctx, file_path)
+    ...
 
-def app() -> None:
```
Comment (jo-jstrm) [inline device-identifier/shbdeviceidentifier/app.py:18]: Do we not need `app()`anymore?

**Review by jo-jstrm — APPROVED** [review]
If we do not need app() anymore, it looks great!

File: device-identifier/shbdeviceidentifier/app.py (hunk 1 above)
Comment (TB-DevAcc) [reply device-identifier/shbdeviceidentifier/app.py:18]: Has been replaced by line 130 in the new file. Still there.

**Pull request merged**